src/screenshots/index.js

```diff
@@ -1,30 +1,58 @@
-import { join as joinPath } from 'path';
-import shortId from 'shortid';
 import { find } from 'lodash';
 import sanitize from 'sanitize-filename';
+import dateFormat from 'dateformat';
 import Capturer from './capturer';
 
 export default class Screenshots {
     constructor (path) {
         this.enabled         = !!path;
         this.screenshotsPath = path;
         this.testEntries     = [];
+        this.startDate       = Screenshots._getStartTestDate();
+        this.userAgentNames  = [];
+    }
+
+    static _getStartTestDate () {
```

Comment:
Also it's not quite correct (and `startDate` var name as well) - it's not a date object, it's formatted date and more likely it's semantically `baseDir` or something like that.